### ISSUE TITLE
nvda stops reading a text after a long time of reading

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#9082]
### Steps to reproduce:
Open a long text in Notepad, the text I'm testing runs about 29,000 words.
Press nvda + the notebook layout to start playback.
### Actual behavior:
After approximately 20 minutes of reading the nvda interrupts the speech and informs list 1 of 3.
At this point it seems that some screen was displayed more if I preciono the arrows or nvda + a nvda is still inside the notepad.
### Expected behavior:
nvda should be able to continue reading until the end of the text, even if the text is too long.
In 3 attempts to replicate the problem I reproduced in 2 times, once the nvda read the text to the end without problems.
The notebook dims the screen more does not enter standby, ibernation, and also does not lock, there is also no rest screen.
I played with only the notebook and the vocalizer and speak synthesizers.
#### NVDA installed/portable/running from source:
install
#### NVDA version:
alfa 16448
#### Windows version:
10 17763.194
#### Name and version of other software in use when reproducing the issue:
notepad
#### Does the issue still occur after restarting your PC?
yes
#### Have you tried any other versions of NVDA?
no
the long text and the log.
[notepad.txt](https://github.com/nvaccess/nvda/files/2684104/notepad.txt)
[Veja 05.12.2018.txt](https://github.com/nvaccess/nvda/files/2684105/Veja.05.12.2018.txt)

**Comment by larry801** [comment]
It seems that something pop up and interrupted your say all command. Some
application with pop up advertisement or notification cause similar
problems on my laptop.
By the way According to this part of your log, some outdated addons may
also cause this problem.
objloc need an update to keep up with new NVDA api
WARNING - eventHandler._EventExecuter.next (21:55:27.490):
Could not execute function event_becomeNavigatorObject defined in
globalPlugins.objloc module due to unsupported kwargs: {'isFocus': False}

**Comment by Adriani90** [comment]
@fernando-jose-silva are you having this problem also when disabling all addons?

**Comment by fernando-jose-silva** [comment]
By my tests the culprit is the dropbox, which is stealing the focus.
The problem has not been solved because the dropbox continues with this behavior, however the problem is not of nvda and can be closed.

**Comment by ehollig** [comment]
@fernando-jose-silva thank you for reporting back. Closing
